src/hooks/useArrDataForBlog.js

```diff
@@ -0,0 +1,70 @@
+import card_header from '../assets/card_header.svg';
+
```

Comment:
Bardzo fajny wstęp do custom hooka :) Co prawda są tu tylko informacje a nie ma jeszcze logiki, ale co do ogólnego zastosowania i importu custom hooka jest to dobry kierunek